test/index.ts

```diff
@@ -601,21 +610,8 @@ describe('Storage', () => {
       );
     });
 
-    it('should make request with method options as query parameter', async () => {
```

Comment:
the logic in this test is duplicated by the logic in the "should make correct API request" test, so this test can be safely removed.